python/lancedb/remote/table.py

```diff
@@ -31,7 +32,12 @@ def __repr__(self) -> str:
         return f"RemoteTable({self._conn.db_name}.{self.name})"
 
     def schema(self) -> pa.Schema:
```

Comment:
let's cache this? we don't need one schema call per table version

Reply:
Done